tests/test_epic.py

```diff
@@ -2,6 +2,7 @@
 from selection.index import Index
 from selection.workload import Column, Query, Table, Workload
 
+import sys
```

Comment:
These and the following changes can probably be ignored because the testing must be changed to your testing system anyway.